kanbananza-backend/src/main/services/card.js

```diff
@@ -15,6 +15,11 @@ const createCard = async ({ name, columnId, order, description, priority }) => {
     }
   });
 
+  if (order === undefined) {
```

Comment:
I think if cardsOfColumn was ordered you can just use the last one, so it's more optimal.

Reply:
probably true but its not a priority